src/sentry/feedback/usecases/create_feedback.py

```diff
@@ -142,8 +142,14 @@ def fix_for_issue_platform(event_data):
     # If no user email was provided specify the contact-email as the user-email.
     feedback_obj = event_data.get("contexts", {}).get("feedback", {})
     contact_email = feedback_obj.get("contact_email")
-    if not ret_event["user"].get("email", ""):
-        ret_event["user"]["email"] = contact_email
+
+    if contact_email:
+        if not ret_event["user"].get("email", ""):
+            ret_event["user"]["email"] = contact_email
+
+        # Set the user.email tag
```

Comment:
can we mention in the comment why we're doing this manually

Reply:
yup